servers/src/test/java/tachyon/worker/block/BlockMetadataManagerTest.java

```diff
@@ -216,7 +266,7 @@ public void moveBlockMetaTest() throws Exception {
   }
 
   @Test
-  public void moveBlockMetaExceedCapacity() throws Exception {
+  public void moveBlockMetaExceedCapacityDeprecated() throws Exception {
```

Comment:
As the original `moveBlockMeta(BlockMeta , BlockStoreLocation)` method was **Deprecated** and  the new one `moveBlockMeta(BlockMeta, TempBlockMeta)` has **not been covered** in this UT class. So I prefer to add new UT for new one and suffix the old API's UT methods' names to **distinguish** them better :)